packages/static-hosting/lib/static-hosting.ts

```diff
@@ -495,7 +495,7 @@ export class StaticHosting extends Construct {
     // If the remap is to a different path, create a Lambda@Edge function to handle this
     if (from !== to) {
       // Remove special characters from path
-      const id = from.replace(/[&\/\\#,+()$~%'":*?<>{}]/g, "-");
+      const id = from.replace(/[&/\\#,+()$~%'":*?<>{}]/g, "-");
```

Comment:
@krishanthisera is this expected?

Reply:
This is expected, again unnecessary escape character